Source/Mock/lhc_mock.cpp

```diff
@@ -108,3 +124,38 @@ bool Mock_Internal_HCHttpCallPerformAsync(
 
     return true;
 }
+
+HRESULT Mock_Internal_ReadRequestBodyIntoMemory(
+    _In_ HCCallHandle originalCall,
+    _Out_ http_internal_vector<uint8_t>* bodyBytes
+    )
+{
+    HCHttpCallRequestBodyReadFunction readFunction;
```

Comment:
init all of these

Reply:
we return if they won't be set, is init important? i don't have a huge problem with it, but curious